### DISCUSSION TITLE
Github Student Pack Re-verification mail?

### DISCUSSION THREAD

**Discussion by brakhage33** [community/community#114928] [GitHub Education]
### Hi everyone,

Almost a month ago i received a mail from github that said (a few days after getting mi student benefits confirmed) that i needed to re-verify those student benefits within 30 days. When i go to the link it just tells me when my last verification was, and that i don´t need to do anything (it won´t even let me). Was the mail sent by error or does this have to do with a problem whithin the /benefits page?

**Comment by samus-aran** [comment]
Hi @aMapOfTheStars 👋 

It appears that you have since received your academic benefits so I suspect it was sent in error. 👍

**Comment by 4ksr** [comment]
> ### Hi everyone,
> 
> Almost a month ago i received a mail from github that said (a few days after getting mi student benefits confirmed) that i needed to re-verify those student benefits within 30 days. When i go to the link it just tells me when my last verification was, and that i don´t need to do anything (it won´t even let me). Was the mail sent by error or does this have to do with a problem whithin the /benefits page?

# Response

Hi,

Thank you for reaching out about your GitHub student benefits verification.

It's possible that the email you received was sent in error, or there may have been a temporary issue with the benefits verification process on the /benefits page.

To ensure that your student benefits are properly verified, I recommend reaching out to GitHub Support directly for assistance. They will be able to provide you with personalized support and help resolve any issues or confusion regarding your student benefits verification.

You can contact GitHub Support through the GitHub Help page or by submitting a support ticket via the GitHub Support portal. They'll be able to investigate the situation further and provide you with guidance on how to proceed.

If you have any further questions or concerns, please feel free to ask. We're here to help!

Best regards,
ksr
